packages/store/src/StoreCore.sol

```diff
@@ -239,41 +238,48 @@ library StoreCore {
     Schema schema
   ) internal view returns (bytes memory) {
     // Get the static data length
-    uint256 outputLength = schema.staticDataLength();
+    uint256 staticLength = schema.staticDataLength();
+    uint256 outputLength = staticLength;
 
     // Load the dynamic data length if there are dynamic fields
     PackedCounter dynamicDataLength;
     uint256 numDynamicFields = schema.numDynamicFields();
     if (numDynamicFields > 0) {
       dynamicDataLength = StoreCoreInternal._loadEncodedDynamicDataLength(table, key);
-      outputLength += 32 + dynamicDataLength.total(); // encoded length + data
+      // TODO should total output include dynamic data length even if it's 0?
```

Comment:
Currently none, this was just something unexpected I noticed. This TODO would be well-resolved by this behaviour just being documented.
I suppose in decoders this could be of some unimportant inconvenience